JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -2904,6 +2927,18 @@
 .mmMessageANOVAtype     <- function(type) {
   gettextf("Type %s Sum of Squares",type)
 }
+.mmMessageREgrouping    <- function(RE_grouping_factors) {
+  gettextf(
+    "The following %s used %s random effects grouping %s: %s.",
+    ifelse(length(RE_grouping_factors) < 2, "variable is", "variables are"),
+    ifelse(length(RE_grouping_factors) < 2, "as a", "as"),
+    ifelse(length(RE_grouping_factors) < 2, "factor", "factors"),
+    ifelse(length(RE_grouping_factors) == 0, "...", paste0("'", RE_grouping_factors, "'", collapse = ", "))
+  )
+}
```

Comment:
Use `sprintf(ngettext()` for this. Aside from the strings not being in `gettext()` the structure of this sentence might need to be entirely different in another language.

Reply:
Yeah, thanks. I keep forgetting those.

What do you advise to do about it then?

Reply:
I could maybe create two complete sentences for each case

Reply:
Something like:
```
sprintf(ngettext(length(RE_grouping_factors) < 2,
                 "The following variable is used as random effects grouping factor: %s.",
                 "The following variables are used as random effects grouping factors: %s."),
        ifelse(length(RE_grouping_factors) == 0, "...", paste0("'", RE_grouping_factors, "'", collapse = ", ")))
```

untested though